docs/src/main/docs/microprofile/07_tracing.adoc

```diff
@@ -0,0 +1,108 @@
+///////////////////////////////////////////////////////////////////////////////
+
+    Copyright (c) 2018,2019 Oracle and/or its affiliates. All rights reserved.
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+        http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+///////////////////////////////////////////////////////////////////////////////
+
+= Tracing
+:description: Helidon MP Tracing Support
+= :keywords: helidon, tracing, microprofile, micro-profile
+
+== Configuring Tracing with Helidon MP
+Tracing support is implemented for both for Helidon MP Server and for Jersey client.
+
+Declare the following dependency in your project:
+
+[source,xml]
+----
+<dependency>
+    <groupId>io.helidon.microprofile.tracing</groupId>
+    <artifactId>helidon-microprofile-tracing</artifactId>
+</dependency>
+----
+
+In addition you need to add one of the tracer implementations (Zipkin or Jaeger).
+
+The minimal required configuration is `tracing.service` that contains
+the service name to be associated with tracing spans sent by this instance.
+
+All tracer specific configuration is expected in config under `tracing`.
+
+Example `microprofile-config.properties` with minimal tracer configuration:
+
+----
+tracing.service=helidon-mp
+----
+
+For additional supported properties, please see <<tracing/01_tracing.adoc#Tracing-config,tracing configuration>>
+
+== Creating custom spans
+Microprofile OpenTracing implementation will add support to simply
+add custom spans by annotation. Until we implement this support, you
+can configure custom spans as follows (in JAX-RS resources):
+
+[source,java]
+----
+@Context
+io.helidon.webserver.ServerRequest serverRequest;
+
+//...
+
+Span span = GlobalTracer.get()
+        .buildSpan("my-operation")
+        .asChildOf(serverRequest.spanContext())
+        .start();
+----
```

Comment:
With the recent activation change, the new span should become part of the server request span, right? Perhaps we can add a note to that. I haven't actually tried this with MP, though.